<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>446</width>
                <height>184</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>阈值设置</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>441</width>
                    <height>181</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <layout class="QGridLayout" name="gridLayout">
                                <item row="2" column="1">
                                    <widget class="QLineEdit" name="SizeEdit">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                                <horstretch>0</horstretch>
                                                <verstretch>1</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                        <property name="dragEnabled">
                                            <bool>true</bool>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="0">
                                    <widget class="QLabel" name="label_2">
                                        <property name="text">
                                            <string>Conf</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="0">
                                    <widget class="QLabel" name="label">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>120</width>
                                                <height>16777215</height>
                                            </size>
                                        </property>
                                        <property name="contextMenuPolicy">
                                            <enum>Qt::DefaultContextMenu</enum>
                                        </property>
                                        <property name="layoutDirection">
                                            <enum>Qt::RightToLeft</enum>
                                        </property>
                                        <property name="text">
                                            <string>IoU</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                        <property name="indent">
                                            <number>0</number>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="1">
                                    <widget class="QLineEdit" name="IoUEdit">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                                <horstretch>0</horstretch>
                                                <verstretch>1</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="layoutDirection">
                                            <enum>Qt::LeftToRight</enum>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="0">
                                    <widget class="QLabel" name="label_3">
                                        <property name="text">
                                            <string>图像大小</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="1">
                                    <widget class="QLineEdit" name="ConfEdit">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                                <horstretch>0</horstretch>
                                                <verstretch>1</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="SaveButton">
                                <property name="text">
                                    <string>Save</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="CancelButton">
                                <property name="text">
                                    <string>Cancel</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
    </widget>
    <tabstops>
        <tabstop>IoUEdit</tabstop>
        <tabstop>ConfEdit</tabstop>
        <tabstop>SizeEdit</tabstop>
        <tabstop>SaveButton</tabstop>
        <tabstop>CancelButton</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
